recipe/meta.yaml

```diff
@@ -28,7 +28,7 @@ test:
     - gsw
 
 about:
-  home: https://github.com/TEOS-10/python-gsw/
+  home: https://github.com/TEOS-10/GSW-Python/
```

Comment:
@efiring even though I merged https://github.com/conda-forge/gsw-feedstock/pull/8 that modification would get overwritten when re-rendering the feedstock. The README in conda-forge is auto-generated with `conda-smithy` so we need to change only the metadata in the recipe.